wire-ios-data-model/Source/Universal backup/Model/ConversationBackupModel.swift

```diff
@@ -18,22 +18,35 @@
 
 import Foundation
 
-public struct ConversationBackupModel {
+public struct ConversationBackupModel: Codable {
 
-    public let qualifiedID: QualifiedID
+    public let id: UUID
 
-    public let name: String
+    public let domain: String
 
-    public let participants: Set<QualifiedID>
+    public let name: String?
+
+    public let otherParticipants: Set<QualifiedID>?
```

Comment:
What is the difference between an empty set and nil?

Reply:
Same here, we'll delete this